server/adaptors/integrations/__data__/repository/nginx/nginx-1.0.0.json

```diff
@@ -7,7 +7,7 @@
     "type": "logs",
     "link": "https://www.nginx.com/",
     "author": "OpenSearch",
-    "sourceUrl": "https://github.com/opensearch-project/observability/tree/2.x/integrations/nginx",
+    "sourceUrl": "https://github.com/opensearch-project/dashboards-observability/tree/main/server/adaptors/integrations/__data__/repository/nginx",
```

Comment:
Slight inconsistency with linking the integration root here and `info` in VPC, but not a big deal. I think my ideal solution is avoid using an info directory entirely and throw the readme in the integration root, but it probably doesn't matter much